### PR TITLE
New site, but Zola

### PR DESCRIPTION
**Pull request by victorbnl**
**TL;DR:** It's #14 but in Zola

Honestly it's true that the website isn't very attractive, and I think it would be bad not having a good site for such an amazing app! So I looked in the pull requests and saw #14 which I really liked, and I saw you didn't approve it because it uses Hugo instead of Zola. So I decided to port his site in Zola and here it is

image

### CODE REVIEW

**Comment by biodranik** [comment]
Thanks! The general idea is good, but there are some layout issues that definitely need to be fixed.

1. Please try to bind your site to CloudFlare pages or autobuild it with your GH pages, so everyone can easily see and comment on the site preview.
2. The dark theme was deleted.
3. There are many layout issues if you make a low-height landscape site.
4. As I already mentioned in the closed PR, it's better to show several screenshots in the landscape mode, instead of waiting while one screen in the center will change.
5. Mobile layout also needs love and tuning. The idea with a phone frame is ok in portrait, but it should be enlarged to take almost all screen space (otherwise the picture is too small and unreadable).

**Comment by victorbnl** [comment]
I think that I'll create a whole new site then, because it will be hard to edit another one's code, and I don't agree neither with all his programming choices etc so it'd be easier to just do it again from scratch, using his beautiful site as a base.

**Comment by biodranik** [comment]
The hardest part is to properly layout everything for all devices. It's plain HTML/markdown and CSS, so the programming choice does not matter much. Taking our code as a base and modifying it step by step may be a better idea than throwing everything out.

**Comment by victorbnl** [comment]
Then I'll also use your code.
Also would it be possible to have a SVG of your logo? If not don't worry, I can vectorize it in Illustrator

**Comment by biodranik** [comment]
I've uploaded logos into the static folder.

@rtsisyk we need to insert a visible and recognizable logo somewhere :) Any ideas?

**Comment by victorbnl** [comment]
Ok so I made the new site

1. https://organicmaps-github-io.vercel.app/
2. Added back
3. Not anymore
4. As you can see it's part of added but fully bugged, that's the part I need to work on
5. Not the case anymore

As you can see I didn't fully replicate his site but customized it a bit, I felt like it missed colors so I made the header green and material design. Tell me if you don't like it or if you'd like to change some things!

**Comment by victorbnl** [comment]
image

**Comment by maltejur** [comment]
I can't scroll further on the screenshots right now
screenshot

**Comment by victorbnl** [comment]
> I can't scroll further on the screenshots right now
> screenshot

You're right, it should be fixed now

File: sass/index.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  &-wrapper {
+    overflow-x: auto;
+    &-wrapper {
+      width: 75vw;
+      position: relative;
+      left: calc(-38vw + 50%);
+      display: flex;
+      justify-content: center;
+      margin-bottom: 4em;
```
Comment (maltejur) [inline sass/index.scss:23]:
```suggestion
      margin-bottom: 4em;
      
      @media (max-width: 800px) {
        width: 100%;
        left: 0;
      }
```

Comment (maltejur) [reply sass/index.scss:23]: There is a lot of white space on mobile, maybe something like this?

Comment (victorbnl) [reply sass/index.scss:23]: Yep I agree

**Comment by maltejur** [comment]
The color of the titles is slightly off, is that intentional?

**Comment by victorbnl** [comment]
Yes, when I tested I saw it was difficult to read it on a light theme with the exact accent color

**Comment by victorbnl** [comment]
I tested it again and saw I was wrong ; I'll put the real accent color intstead

**Comment by victorbnl** [comment]
It's difficult to distinguish the titles and the links, but I don't know how to solve it. Adding the underline again? A slightly different color?

image

**Comment by maltejur** [comment]
Change the color to the lighter green, but only when in dark mode

**Comment by victorbnl** [comment]
> Change the color to the lighter green, but only when in dark mode

I did that yes, but I was not talking about distinguishing the titles and the links from the background of the page, but distinguishing the links from the titles, it's difficult as they look the same

**Comment by maltejur** [comment]
Ahh, then I'd just add the underline to the links again

**Comment by biodranik** [comment]
Looks better now. Issues:
1. Top screen space is used inefficiently: there is a lot of free space, the title "Organic Maps" is duplicated.
2. Stores buttons are of different sizes.
3. Screenshots should wrap depending on the width, without scroll (it's unusable).
4. Screenshots will look better and more readable if their size will be bigger (ideally the same or similar as it's now).
5. Footer text should be smaller.

**Comment by victorbnl** [comment]
1. [ ] Top screen space is used inefficiently: there is a lot of free space, the title "Organic Maps" is duplicated.
2. [ ] Stores buttons are of different sizes.
3. [ ] Screenshots should wrap depending on the width, without scroll (it's unusable).
4. [ ] Screenshots will look better and more readable if their size will be bigger (ideally the same or similar as it's now).
5. [ ] Footer text should be smaller.
6. [ ] Please also rebase your PR or resolve conflict by merging the updated README.

**Comment by victorbnl** [comment]
For the "Organic Maps" repeated twice I don't think it's a problem, it would be really strange to just start with "Offline Hike, Bike, Trails and Navigation" without a title for me (I think the title in the header doesn't really count as it's not in the content)

For the screenshots which should not scroll, maybe, but I think wrapping them would be a bad idea, it would take years scrolling to finally get to the content

**Comment by victorbnl** [comment]
For the badges, there are rules to use it and you can't resize them or re-arrange elements as you want so I don't think it's authorized to use unofficial ones. And official App Store and Google Play badges' width differ so...

**Comment by biodranik** [comment]
image

This is not usable.

Badges may and should be of the equal size, it's our site and we want it to look great.

Screenshots are the best way to quickly demonstrate features. Current version proves it.

**Comment by victorbnl** [comment]
> This looks very strange.

It's true that on your screen there is too much top space, but for the title repeated twice I don't see any problem

> Badges may and should be of the equal size, it's our site and we want it to look great.

Alright then, I'll fix that

> Screenshots are the best way to quickly demonstrate features. Current version proves it.

I agree but I don't know if putting them like this vertically is the best way. But I'll try it anwyay

**Comment by victorbnl** [comment]
The badges should now be of the same size

**Comment by victorbnl** [comment]
Now it should do what you want for the screenshots

**Comment by victorbnl** [comment]
I think I fixed all the points you said

**Comment by victorbnl** [comment]
Also do you want me to rebase and sign off my commits to make DCO succeed?

**Comment by vitaly-zdanevich** [comment]
Maybe increase font size for popular 1920x1080?

image

**Comment by victorbnl** [comment]
Ah yes, I work on a laptop so I kind of forget this some times x)
I can't today but I'll fix that tomorrow (or maybe this evening)

**Comment by victorbnl** [comment]
It should now be fixed

**Comment by biodranik** [comment]
As I already mentioned in another PR, I do not understand why you deleted all existing files and introduced a completely different file structure, instead of modifying CSS and HTML templates for a new look.

The top part also needs to be improved:
1. I would reduce the space from the top bar to the heading.
2. I would prefer to append the second heading to the main one because Organic Maps is already visible in the top left.
3. We want to insert the FAQ page there, and later few more pages. How it will look in your design?

**Comment by victorbnl** [comment]
> As I already mentioned in another PR, I do not understand why you deleted all existing files and introduced a completely different file structure, instead of modifying CSS and HTML templates for a new look.

Yes I'm sorry for that

> We want to insert the FAQ page there, and later few more pages. How it will look in your design?

You'll be able to add any markdown content, it will look like the privacy and terms page basically

**Comment by victorbnl** [comment]
> I would prefer to append the second heading to the main one because Organic Maps is already visible in the top left.

Like `Organic Maps: Offline Hike, Bike, Trails and Navigation` ?

**Comment by victorbnl** [comment]
I reduced the top space, and made it smaller on small screens

**Comment by biodranik** [comment]
The fixed top menu consumes important vertical space on small screens. It's better to make a normal scrollable header/menu where FAQ and language switcher can be inserted later.

**Comment by victorbnl** [comment]
Ah you want the menu to be in the normal page flow so that it  disappears when scrolling?

**Pull request closed without merging**